### ISSUE TITLE
Failed to start node-agent

### ISSUE THREAD

**Issue by oskoi** [coroot/coroot-node-agent#151]
version: **v1.22.5**

logs:
```
I1208 12:17:03.946190 3028419 net.go:23] whitelisted public IPs: [0.0.0.0/0]
I1208 12:17:03.946294 3028419 net.go:35] ephemeral-port-range: 32768-60999
I1208 12:17:03.954701 3028419 cilium.go:30] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_ct4_global: no such file or directory
I1208 12:17:03.954749 3028419 cilium.go:36] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_ct6_global: no such file or directory
I1208 12:17:03.954770 3028419 cilium.go:43] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_lb4_backends_v2: no such file or directory
I1208 12:17:03.954786 3028419 cilium.go:43] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_lb4_backends_v3: no such file or directory
I1208 12:17:03.954829 3028419 cilium.go:52] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_lb6_backends_v2: no such file or directory
I1208 12:17:03.954849 3028419 cilium.go:52] Unable to get object /proc/1/root/sys/fs/bpf/tc/globals/cilium_lb6_backends_v3: no such file or directory
I1208 12:17:03.960791 3028419 main.go:108] agent version: 1.22.5
I1208 12:17:03.961010 3028419 main.go:114] hostname: gke-burny-ops-us-ea4-argocd-717f6aac-3cik
I1208 12:17:03.961027 3028419 main.go:115] kernel version: 6.1.100+
I1208 12:17:03.961864 3028419 main.go:72] machine-id:  57012b0317b98347ba69469dee6bc61f
I1208 12:17:03.962098 3028419 tracing.go:40] OpenTelemetry traces collector endpoint: http://coroot-opentelemetry-collector:4318/v1/traces
I1208 12:17:03.962285 3028419 otel.go:30] OpenTelemetry logs collector endpoint: http://coroot-opentelemetry-collector:4318/v1/logs
I1208 12:17:03.962654 3028419 metadata.go:67] cloud provider: GCP
I1208 12:17:03.966578 3028419 collector.go:157] instance metadata: &{Provider:GCP AccountId:appglider InstanceId:6027213135274638061 InstanceType:e2-medium LifeCycle:on-demand Region:us-east4 AvailabilityZone:us-east4-a AvailabilityZoneId: LocalIPv4:10.150.0.119 PublicIPv4:}
I1208 12:17:03.967001 3028419 profiling.go:55] profiles endpoint: http://coroot:8080/v1/profiles
I1208 12:17:07.364086 3028419 containerd.go:38] using /run/containerd/containerd.sock
I1208 12:17:07.364166 3028419 crio.go:58] cri-o socket:
F1208 12:17:08.350141 3028419 main.go:146] failed to link program 'nf_conntrack_hash_insert': creating perf_kprobe PMU (arch-specific fallback for "__nf_conntrack_hash_insert"): token __x64___nf_conntrack_hash_insert: not found: no such file or directory
F1208 12:17:08.350141 3028419 main.go:146] failed to link program 'nf_conntrack_hash_insert': creating perf_kprobe PMU (arch-specific fallback for "__nf_conntrack_hash_insert"): token __x64___nf_conntrack_hash_insert: not found: no such file or directory
F1208 12:17:08.350141 3028419 main.go:146] failed to link program 'nf_conntrack_hash_insert': creating perf_kprobe PMU (arch-specific fallback for "__nf_conntrack_hash_insert"): token __x64___nf_conntrack_hash_insert: not found: no such file or directory
F1208 12:17:08.350141 3028419 main.go:146] failed to link program 'nf_conntrack_hash_insert': creating perf_kprobe PMU (arch-specific fallback for "__nf_conntrack_hash_insert"): token __x64___nf_conntrack_hash_insert: not found: no such file or directory
F1208 12:17:08.350141 3028419 main.go:146] failed to link program 'nf_conntrack_hash_insert': creating perf_kprobe PMU (arch-specific fallback for "__nf_conntrack_hash_insert"): token __x64___nf_conntrack_hash_insert: not found: no such file or directory
```

**Comment by StLeoX** [comment]
Did `conntrack` install on your OS?
Check it: `dpkg -l|grep conntrack`

**Comment by def** [comment]
@oskoi thanks for the report. This bug is occurring on kernels without DEBUG symbols. We’re working on a fix. We've built the 1.22.7 with some changes reverted

**Comment by def** [comment]
this is fixed in v1.22.8
